src/lib/y2partitioner/widgets/format_and_mount.rb

```diff
@@ -183,7 +204,7 @@ def refresh
         if filesystem
           Yast::UI.ChangeWidget(Id(:mount_device), :Enabled, true)
 
-          if filesystem.mountpoint.nil? || filesystem.mountpoint.empty?
+          if filesystem.mount_path.nil? || filesystem.mount_path.empty?
```

Comment:
If we need that more often, how about a function `mount_path?` that checks if non-nil and non-empty?

Reply:
I think it is not needed. In fact, that line can be replaced by `filesystem.mount_point.nil?`